docs/internals/contributing/triaging-tickets.txt

```diff
@@ -460,7 +460,7 @@ find a revision where your test passes, mark it as "good"::
 Now we're ready for the fun part: using ``git bisect run`` to automate the rest
 of the process::
 
-    $ git bisect run python runtests.py migrations.test_regression
+    $ git bisect run PYTHONPATH=tests tests/runtests.py migrations.test_regression
```

Comment:
That's true. The only reason to change the `PYTHONPATH` it is to keep the test command from the explanations above, and possibly the path to the `--settings`. It's mainly out of convenience, but it might be too specific.

I don't have a strong opinion on this, tell me what you prefer and I will update the PR.